user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/GQIColumn.md

```diff
@@ -1,12 +1,31 @@
 ---
-uid: GQIColumn
+uid: GQI_GQIColumn
 ---
 
-# GQIColumn
+# GQIColumn Class
 
-The *GQIColumn* object is an abstract class, with the derived types *GQIStringColumn*, *GQIBooleanColumn*, *GQIIntColumn*, *GQIDateTimeColumn*, *GQIDoubleColumn*, and (from DataMiner 10.3.9/10.4.0 onwards<!-- RN 36717 -->) *GQITimeSpanColumn*, and with the following properties:
+## Definition
+
+Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
+Assembly: `SLAnalyticsTypes.dll`
+
+Provides the base class for a column in GQI.
+
+## Derived types
+
+The *GQIColumn* object is an abstract class, with the derived types:
+- `GQIStringColumn`
+- `GQIBooleanColumn`
+- `GQIIntColumn`
+- `GQIDateTimeColumn`
+- `GQIDoubleColumn`
+- `GQITimeSpanColumn` (from DataMiner 10.3.9/10.4.0 onwards<!-- RN 36717 -->)
+
+## Properties
 
 | Property | Type | Required | Description |
 |--|--|--|--|
 | Name | String | Yes | The column name. |
 | Type | GQIColumnType | Yes | The type of data in the column. *GQIColumnType* is an enum that contains the following values: String, Int, DateTime, Boolean, Double, or TimeSpan. |
+
+`GQIColumnType` is an enum that contains the following values: `String`, `Int`, `DateTime`, `Boolean`, `Double` or `TimeSpan`.
```

Comment:
Duplicate? This explanation is already in the table above.
Use a bullet list to list the enum values?